.github/workflows/go_generate.yml

```diff
@@ -25,13 +25,14 @@ jobs:
           fetch-depth: 0
 
       - name: check 'go generate' is clean
-        # The shell script invocation below is a temporary hack for
-        # https://github.com/tailscale/tailscale/issues/4194. When
-        # that issue is fixed, replace its invocation with:
-        # go generate --tags=hermetic ./...
         run: |
-          set -e
-          ./.github/workflows/go-generate-without-stringer.sh
+          if [[ "${{github.ref}}" == release-branch/* ]]
+          then
+            pkgs=$(go list ./... | grep -v dnsfallback)
+          else
+            pkgs=$(go list ./... | grep -v dnsfallback)
+          fi
```

Comment:
are these two identical?

Reply:
yes, it's how it was before. It's so we can make them different easily in the future I believe.